website/package.json

```diff
@@ -14,11 +14,11 @@
     "write-heading-ids": "docusaurus write-heading-ids"
   },
   "dependencies": {
-    "@docusaurus/core": "^2.0.0-beta.3",
-    "@docusaurus/plugin-client-redirects": "^2.0.0-beta.3",
-    "@docusaurus/plugin-sitemap": "^2.0.0-beta.3",
-    "@docusaurus/preset-classic": "^2.0.0-beta.3",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.3",
+    "@docusaurus/core": "2.0.0-beta.14",
+    "@docusaurus/plugin-client-redirects": "2.0.0-beta.14",
+    "@docusaurus/plugin-sitemap": "^2.0.0-beta.14",
+    "@docusaurus/preset-classic": "2.0.0-beta.14",
+    "@docusaurus/theme-search-algolia": "^2.0.0-beta.14",
```

Comment:
this one also needs fix. remove "^" at the beginning.

Reply:
I followed the pattern of the original PR @vingov put up (some have fixed version and some have "up to" constraint).  Let me test the latest and freeze the versions altogether.